test/io/testfilewriter.py

```diff
@@ -20,15 +20,30 @@
 
 class WriterTest(unittest.TestCase):
 
+    def setUp(self):
+        self.MissingAttributeError = cclib.io.filewriter.MissingAttributeError
+
     def test_init(self):
         """Does the class initialize correctly?"""
-        fpath = os.path.join(__datadir__, "data/ADF/basicADF2007.01/dvb_gopt.adfout")
+        fpath = os.path.join(__datadir__,
```

Comment:
You don't need to actually parse a file just to test the initializer. You can use an empty ccData object for that, right?